src/Spec2-Core/SpAbstractTextPresenter.class.st

```diff
@@ -311,8 +311,18 @@ SpAbstractTextPresenter >> text [
 { #category : #api }
 SpAbstractTextPresenter >> text: aString [
 	"Set the text of the text presenter"
-
-	text := aString
+	"Attempts to keep text attributes (formatting) of the field"
```

Comment:
this change is actually incorrect.
Spec is not Text aware nor it will be. An attempt to mix it here will just add a functionality that we do not support, and we will not support that way. 
I understand people may want to use a Text in Spec with Morphic (but Spec is agnostic of its backend). 
Instead, I suggest to use create a special presenter that is attached to the Text backend.

Reply:
Thanks Esteban. I understand where Spec is going. It is a reasonable choice to have some widgets which are String only, and some which do character based formatting. I was just not aware of which special presenter can use Text without morphic.

Reply:
just to be clear, there is *not* such presenter right now, you need to create it :)